ui/conductor/src/routes/devices/components/DeviceCell.vue

```diff
@@ -6,37 +6,37 @@
         :name="props.item.name"
         :paused="props.paused">
       <EnterLeaveEventCell
-          v-if="!resource.streamError && (resource?.value?.enterTotal || resource?.value?.leaveTotal)"
-          v-bind="resource"/>
-      <StatusAlert v-else :resource="resource.streamError"/>
+          v-if="resource?.value?.enterTotal || resource?.value?.leaveTotal"
+          v-bind="resource"
+          :stream-error="resource.streamError"/>
```

Comment:
You don't need this `:stream-error` attr. the `v-bind="resource"` already does it as `resource` has a `streamError` property